tensorflow/python/keras/optimizer_v2/optimizer_v2.py

```diff
@@ -445,7 +445,8 @@ def apply_grad_to_update_var(var, grad):
       return apply_updates
 
   def get_updates(self, loss, params):
-    grads = self.get_gradients(loss, params)
+    with backend.get_graph().as_default():
```

Comment:
I think this should be in get_gradients rather than get_updates.

Reply:
Moved it.